libs/vkd3d/state.c

```diff
@@ -5548,7 +5548,7 @@ static void d3d12_pipeline_state_log_graphics_state(const struct d3d12_pipeline_
 
     if (graphics->dsv_format)
     {
-        ERR("DSV: %u\n", graphics->dsv_format);
+        ERR("DSV: %p\n", graphics->dsv_format);
```

Comment:
Should be graphics->dsv_format->dxgi_format, and format should be "DSV: #%x".

Reply:
Is `#%x` an actual format specifier I'm just unaware of or is it just supposed to be unmodified unsigned-int-as-hex with `#` prepended in front if it? I believe `%#x` would be a proper format (unsigned-int-as-hex-with-0x-in-front) but I see in a _lot_ of places `#%x` being used so I'll go with that one for now, though I can't guess the intent here.